### ISSUE TITLE
bidirectional chat not working as expected

### ISSUE THREAD

**Issue by Prabumca12** [Azure/azure-functions-signalrservice-extension#146]
I'm able to connect hub but after there is no communication happened on bidirectional chat

![image](https://user-images.githubusercontent.com/52482192/97863826-a3933700-1d2d-11eb-93b7-40f2bcac36ec.png)

**Comment by zackliu** [comment]
Please uncheck `send To Default Group: AzureSignalR` because you're not in any group.
After you uncheck the checkbox, you can text something in textbox and press enter to broadcast

**Comment by Prabumca12** [comment]
hi zackliu,

I have tried all the possibilities but not working for me and working only with negotiate (http trigger). I assumed signalr trigger not invoked. Is there any configuration missed me for my end?

![image](https://user-images.githubusercontent.com/52482192/99210757-ab061600-27ec-11eb-8fff-61b5c488bdd1.png)
![image](https://user-images.githubusercontent.com/52482192/99210799-c40ec700-27ec-11eb-8b72-95c31b0fd45d.png)

**Comment by zackliu** [comment]
If the trigger was totally not invoked (You can check it from Function's log), I assume the upstream settings in SignalR Service is not correct. And what's the browser console output when you try to send message? If there's no output, it seems to be the upstream setting is not set?
If you can't find out any reason, you can give me you ResourceID and time, I can check more from backend.

**Comment by Prabumca12** [comment]
There is no error in browser console. 

![image](https://user-images.githubusercontent.com/52482192/99619889-67a8e300-2a4a-11eb-97c5-5e51c478db48.png)

**Comment by zackliu** [comment]
I checked the configuration of your resource, and found the upstream setting was empty. Please check the document to add the upstream setting. https://docs.microsoft.com/en-us/azure/azure-signalr/concept-upstream
For a quick setting and sample, see https://github.com/aspnet/AzureSignalR-samples/tree/chenyl/bi-direction/samples/BidirectionChat#deploy-project-to-azure-function
`In the Upstream URL Pattern, fill in the <function-url>/runtime/webhooks/signalr?code=<signalr_extension-key>`

**Comment by Prabumca12** [comment]
Yeah you're correct. I missed that upstream setting. It's working fine. Is there any possibility to downgrade this project to .Net Core 2.2

**Comment by zackliu** [comment]
It should work in 2.2, while you need to downgrade function tool from v3 to v2
